src/scenes/AcceleratorRouter/ParticipantProjects/ProjectDeliverablesView.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Box, useTheme } from '@mui/material';
+
+import Card from 'src/components/common/Card';
+import DeliverablesList from 'src/scenes/DeliverablesRouter/DeliverablesList';
+
+type ProjectDeliverablesViewProps = {
+  projectId: number;
+};
+
+const ProjectDeliverablesView = ({ projectId }: ProjectDeliverablesViewProps) => {
+  const theme = useTheme();
+
+  return (
+    <Card
+      style={{
+        borderRadius: theme.spacing(1),
+        padding: 0,
+      }}
+    >
+      <Box marginX={`-${theme.spacing(4)}`}>
+        <DeliverablesList projectId={projectId} />
```

Comment:
Is it common for us to use negative margins?

Reply:
There's inconsistent margin/padding throughout the app, often applied in components when it should be on the parent. Specifically, the `Page` and `Tabs` components have padding, and removing or making it optional will require adjustments wherever these components are used.

Wasn't sure if I should tackle it in this PR, but I can.